mlflow/utils/process.py

```diff
@@ -75,6 +77,18 @@ def _exec_cmd(
 
     env = env if extra_env is None else {**os.environ, **extra_env}
 
+    if is_in_databricks_runtime():
+        if python_path_env := os.environ.get("PYTHONPATH"):
+            python_paths = python_path_env.split(":")
+            filtered_paths = []
+            for p in python_paths:
+                try:
+                    os.stat(p)
```

Comment:
Can we use [os.access](https://docs.python.org/ja/3/library/os.html#os.access) to remove try-catch and be more explicit?

Reply:
Good idea!